extras/mmu_machine.py

```diff
@@ -293,6 +293,7 @@ def __init__(self, config, mmu):
         self.mmu = mmu
         self.printer = config.get_printer()
         self.reactor = self.printer.get_reactor()
+        self._saved_stepper_kins = {}
```

Comment:
don't need this...left over from one of the earlier failed attempts to get it working